src/DssSpell.sol

```diff
@@ -54,32 +63,37 @@ contract DssSpellAction is DssAction, DssSpellCollateralAction {
     //
 
     function officeHours() public override returns (bool) {
-        return true;
+        return false;
     }
 
     function actions() public override {
 
         // ---------------------------------------------------------------------
         // Includes changes from the DssSpellCollateralAction
-        onboardNewCollaterals();
+        // onboardNewCollaterals();
         // offboardCollaterals();
 
-        drawFromRWA009Urn();
+        address RWA009_A_URN = DssExecLib.getChangelogAddress("RWA009_A_URN");
+        address MCD_VEST_DAI = DssExecLib.getChangelogAddress("MCD_VEST_DAI");
 
-        // Add RWA_TOKEN_FAB to changelog
-        DssExecLib.setChangelogAddress("RWA_TOKEN_FAB", RWA_TOKEN_FAB);
-
-        DssExecLib.setChangelogVersion("1.13.3");
-    }
+        // Huntingdon Valley (HVBank) Vault Drawdown
+        RwaUrnLike(RWA009_A_URN).draw(RWA009_DRAW_AMOUNT);
```

Comment:
This will fail if the previous spell hasn't been cast yet. We need to double-check with govalpha whether this is ok, or if we want to add a conditional here to skip this if it hasn't been onboarded yet.

Reply:
previous spell is already scheduled

Reply:
I think is fine as it is, we can count on the other one will be cast before. And it actually works as a good way to make sure the other one gets executed before.
The only reason why we would want a conditional is if we think the prev spell could be dropped, and as it hasn't been a malicious spell there is not real reason that to happen.

Reply:
Disregard, previous spell was scheduled so we're fine.

Reply:
> we can count on the other one will be cast before. And it actually works as a good way to make sure the other one gets executed before.

We had previously had some discussions with GovAlpha about skipping the draw when it wasn't clear the exec would pass. It's a non-issue now though.